system_manager:
  ros__parameters:
    health_rate_hz: 1.0
